test/cpp/peer_audio_manager_test.cpp

```
@@ -0,0 +1,185 @@
+    std::cout << "Test Normal Seq Accepted: PASSED" << std::endl;
+}
+
+// High uint32 seq values (>= 0x80000000) look negative when read as a signed
+// 32-bit int but are valid unsigned uint32 seq numbers. The JNI layer passes
+// them as jlong and casts to uint32_t via static_cast — this test confirms
+// that the C++ side of that cast path accepts them correctly.
+//
+// We push a small window (5 frames) that spans the 2^31 boundary so the
+// jitter buffer stays well under kJitterMaxDepth (10).
+void testHighUint32SeqAccepted() {
+    PeerAudioManager mgr;
```

Clang (14.0.6)

[warning] variable 'mgr' is not initialized

(cppcoreguidelines-init-variables)